app/views/page/flowHistoryPage.scala.html

```diff
@@ -24,9 +24,14 @@
       <form id="flow-history-form" role="form" method="get" action="@routes.Application.flowHistory()">
         <div class="form-group">
           <label for="form-flow-def-idurl">Flow Definition URL/ID</label>
-          <input type="text" class="form-control" id="form-flow-def-id" name="flow-def-id" placeholder="Flow Definition URL/ID">
+          <input type="text" class="form-control" id="form-flow-def-id" name="flow-def-id" placeholder="Flow Definition URL/ID" value=@flowDefId>
         </div>
-
+        <select class="form-control" name="select-graph-type">
```

Comment:
As we discussed, I think we should have a drop down to select either Heuristics or Metrics and have a separate checkbox or dropdown for the graph selector in the graph layout.

Also, make sure the default value of the dropdown selector is updated after submitting the form.